tests/sandbox/message_batches/create_message_batches/test_field_validation.py

```diff
@@ -549,6 +549,51 @@ def test_null_personalisation(nhsd_apim_proxy_url, correlation_id, personalisati
     )
 
 
+@pytest.mark.sandboxtest
+@pytest.mark.parametrize("correlation_id", constants.CORRELATION_ID)
+def test_not_permitted_to_use_contact_details(nhsd_apim_proxy_url, correlation_id):
+    """
+    .. include:: ../../partials/validation/test_not_permitted_to_use_contact_details.rst
+    """
+    resp = requests.post(f"{nhsd_apim_proxy_url}{MESSAGE_BATCHES_ENDPOINT}", headers={
+        **headers,
+        "X-Correlation-Id": correlation_id,
+        "Authorization": "notAllowedContactDetailOverride"
+    }, json={
```

Comment:
A while back, Ada made a suggestion on how we can improve the quality of our tests by taking a valid message body and replacing a given attribute with the information that we want to change in order to generate the expected outcome/error, so rather than fully declare the request body in each test we would do something similar to the following:

{code}
    data = Generators.generate_valid_create_message_batch_body("dev")
    data["data"]["attributes"]["messages"][0]["recipient"]["nhsNumber"] = nhs_number
{code}

That way it makes it a bit more obvious what it is that's generating the expected response and overall reduces the total number of lines needed for a given test